### PR TITLE
Add CS and update to upstream 1.2.6

### PR DESCRIPTION
**Pull request by Kdecherf**

### CODE REVIEW

**Comment by jtojnar** [comment]
Thanks, looks good. I can also add you as an admin if you wish. cc @j0k3r

**Comment by j0k3r** [comment]
Won't this be hard to keep the projet update with the upstream if we apply PHP-CS-Fixer?

**Comment by jtojnar** [comment]
Updating should not be a problem – just overwrite the code with fresh upstream one, run php-cs-fixer, and then re-apply our patches that have not been upstreamed yet.

Pushing changes upstream will be harder. Perhaps we could try to convince upstream to use code formatting to make that easier?

**Comment by Kdecherf** [comment]
I manually applied changes shown by a simple diff against archives from sourceforge (`diff -u {1.2.4.2,1.2.6}/htmLawed.php`). Not ideal but it seems that there's a very low activity on upstream anyway.

**Comment by Kdecherf** [comment]
... and fixing things (_outside of upstream patches_) is quite horrible without some CS

**Comment by Kdecherf** [comment]
@jtojnar fine for me for the admin permission :)

**Review by j0k3r — APPROVED** [review]

**Comment by j0k3r** [comment]
@Kdecherf maybe you can add a GitHub workflow to check CS for PR?

**Comment by Kdecherf** [comment]
@j0k3r I've added a workflow but it seems that it will run only after being merged in master

File: composer.json [hunk 1]
```diff
@@ -17,10 +17,16 @@
...
+        "friendsofphp/php-cs-fixer": "^3.0"
+    },
     "replace":{
         "htmlawed/htmlawed": "*"
     },
      "autoload": {
         "files": ["htmLawed.php"]
+    },
+    "config": {
+        "bin-dir": "bin"
```
Comment (jtojnar) [inline composer.json]: Why not just keep it in the `vendor/bin` directory?

Comment (Kdecherf) [reply composer.json]: Yeah, we can just keep the original one

Comment (Kdecherf) [reply composer.json]: done

**Review by j0k3r — COMMENTED** [review]
Looks great, you can squash :+1:

**Pull request merged**